odoo-shippable/scripts/build-image.sh

```diff
@@ -15,6 +15,9 @@ ARCH="$( dpkg --print-architecture )"
 GITCORE_PPA_REPO="deb http://ppa.launchpad.net/git-core/ppa/ubuntu trusty main"
 GITCORE_PPA_KEY="http://keyserver.ubuntu.com:11371/pks/lookup?op=get&search=0xA1715D88E1DF1F24"
 
+# ppa sources
+PPA_SOURCES="fkrull/deadsnakes"
+
```

Comment:
@KarenKawaii The variable, the repo name cannot be changed. My guess is that it don't add dead snakes (JK)...perhaps is better somthing like PYTHON_PPA, PYTHON30_PPA or so